airflow/ui/src/pages/DagRuns.tsx

```diff
@@ -118,17 +117,26 @@ export const DagRuns = () => {
   const { pagination, sorting } = tableURLState;
   const [sort] = sorting;
   const orderBy = sort ? `${sort.desc ? "-" : ""}${sort.id}` : "-run_after";
+  const {
+    END_DATE: END_DATE_PARAM,
+    START_DATE: START_DATE_PARAM,
+    STATE: STATE_PARAM,
+  }: SearchParamsKeysType = SearchParamsKeys;
```

Comment:
We can declare this outside of the component so we don't redeclare it every time the component mounts.